src/main/java/org/terasology/blockNetwork/TwoNetworkNodes.java

```diff
@@ -15,15 +15,32 @@
  */
 package org.terasology.blockNetwork;
 
+    /**
+    * Creates two network nodes, checks whether they are equal to each other or not, and 
+    * calculates the sum of the hashcodes of the locations of both nodes.
+    */
 public class TwoNetworkNodes {
     public final NetworkNode node1;
     public final NetworkNode node2;
-
+    
+    /**
+    * TwoNetWorkNodes constructor.
```

Comment:
this is TwoNetworkNoes and this is a constructor but this does not really explain what this is.

"Instantiates a [class] via two given nodes"